redash/query_runner/duckdb.py

```diff
@@ -0,0 +1,83 @@
+import logging
+import duckdb
```

Comment:
Doesn't this mean there should be a matching "duckdb" entry in one of our requirements*.txt files?

Reply:
Yes there is. I forgot to add that one. Will commit with that change. Thanks.